test/unit/spec/MfaVerify_spec.js

```diff
@@ -1915,6 +1919,14 @@ function (Q,
             expect(test.form.subtitleText()).toBe('Signing into Okta...');
           });
         });
+
+        itp('automatically triggers Windows Hello only once', function () {
+          return emulateWindows()
+          .then(setupWebauthnOnly)
+          .then(function (test) {
+            expect(test.router.controller.model.get('__autoTriggered__')).toBe(true);
```

Comment:
I think what we should try to test is if ```model.save``` is called automatically without the need to press any button.